build/artifacts.props

```diff
@@ -10,9 +10,21 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
+    <!--
+      Exclude the additionalDeps files for this hosting startup in the 2.0.8 release.
+      This should be re-enabled the next time Microsoft.NETCore.App is shipped.
```

Comment:
Can you file a follow up for this and assign it to 2.0.9?

Reply:
Done. I opened #1082 to undo this change in the 2.0.x patch.